lib/irb/command/pushws.rb

```diff
@@ -15,7 +15,7 @@ class Workspaces < Base
       description "Show workspaces."
 
       def execute(*obj)
-        irb_context.workspaces.collect{|ws| ws.main}
+        puts irb_context.instance_variable_get(:@workspace_stack).to_s
```

Comment:
Technically this is a breaking change as now `workspaces.each` would break. But since we say it's a command, uses of its return value is not officially supported IMO.

Reply:
I think breaking change about it is not a big problem 👍 

Maybe `.collect(&:main)` `.map(&:main)` like before?
The `#<IRB::WorkSpace(`  `)>` part is redundant and long, I think.

```
irb> pushws 42
[#<IRB::WorkSpace (main)>, #<IRB::WorkSpace ("a")>, #<IRB::WorkSpace (#<Object:0x00000001...>)>, #<IRB::WorkSpace (42)>]
```